core/src/commonMain/kotlin/info/laht/threekt/controls/FirstPersonControls.kt

```diff
@@ -0,0 +1,240 @@
+package info.laht.threekt.controls
+
+import info.laht.threekt.cameras.Camera
+import info.laht.threekt.core.EventDispatcher
+import info.laht.threekt.core.EventDispatcherImpl
+import info.laht.threekt.input.*
+import info.laht.threekt.math.*
+import kotlin.jvm.JvmOverloads
+import kotlin.math.PI
+import kotlin.math.max
+import kotlin.math.min
+
+class FirstPersonControls(private val camera: Camera, private val eventSource: PeripheralsEventSource) : EventDispatcher by EventDispatcherImpl() {
```

Comment:
Not a fan of long lines like these. Please make the constructor span multiple lines like the rest of the classes in the project.